spec/features/user_can_see_avg_rating_on_book_show_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+describe "As a user" do
+  context "when I go to the books show page" do
+    it "I can see the books average rating" do
+      book = Book.create!(title: "Infinite Jest")
+      rating = Rating.create!(rating: 4, book_id: book.id)
+      rating = Rating.create!(rating: 5, book_id: book.id)
+
+      user = User.create!(name: "Jim")
+
+      visit book_path(book)
+
+      expect(page).to have_content("Average Rating: 4")
```

Comment:
I wouldn't recommend putting wrong information in to make a test pass. Getting the functionality right is more important than passing tests and the functionality isn't right currently.